project-canvas/src/components/backlog-view/Issue/DetailView.tsx

```diff
@@ -0,0 +1,219 @@
+import {
+  Accordion,
+  Avatar,
+  Badge,
+  Box,
+  Button,
+  Grid,
+  Group,
+  Menu,
+  Paper,
+  Stack,
+  Text,
+  ThemeIcon,
+  Title,
+} from "@mantine/core"
+import { IconBinaryTree2 } from "@tabler/icons"
+import { Issue } from "project-extender"
+import { IssueIcon } from "./IssueIcon"
+
+export function DetailView({
+  issueKey,
+  summary,
+  status,
+  storyPointsEstimate,
+  epic,
+  labels,
+  assignee,
+  creator,
+  description,
+  sprintId,
+  subtasks,
+  created,
+  updated,
+  comment,
+  type,
+}: Issue) {
+  return (
+    <Paper p={10}>
+      <Group mb={40} sx={{ gap: "8px" }} color="dimmed" fs="italic">
+        <IssueIcon type="Epic" />{" "}
+        {epic !== undefined ? (
+          <Text>{epic}</Text>
+        ) : (
+          <Text tt="uppercase">Add Epic</Text>
+        )}{" "}
+        /
+        <IssueIcon type={type} /> <Text>{issueKey}</Text>
+      </Group>
+      <Grid>
+        <Box w="60%" sx={{ overflow: "auto", maxHeight: "550px" }}>
```

Comment:
try not to use constant number of px, use standard Mantine spacing values instead where possible and/or % values